adr/0026-schema.md

```diff
@@ -0,0 +1,569 @@
+# 25. Zarf Schema for 1
+
+Date: 2024-06-07
+
+## Status
+
+Proposed
+
+## Context
```

Comment:
I would love to see the context mention how we are basically creating the version schema to set a standard for what we are going to support long term/maintain backwards compat from v1.0 + 
It basically makes the schema a "contract" we have with our users